modules/cnv_controlling_vms.adoc

```diff
@@ -5,15 +5,14 @@
 [[controlvm]]
 = Controlling virtual machines
 
-Virtual machines can be started and stopped, depending on the
-current state of the virtual machine. The option to restart VMs
-is available in the Web Console only.
+You can start, stop, or restart a virtual machine, depending on its current
+state.
 
-Use the *virtctl* client utility to change the state of the virtual
+Use the `virtctl` client utility to change the state of the virtual
 machine, open virtual console sessions with the virtual
 machines, and expose virtual machine ports as services.
 
-The *virtctl* syntax is: `virtctl <action> <VM-name>`
+The `virtctl` syntax is: `virtctl <action> <vm_name> [flags]`
```

Comment:
Why [flags]? I think it should be \<options\>.

Reply:
Ah, I got [flags] from the virtctl --help output. I can definitely change it to <options> if that's what we use.